cloudconfig.tpl

```diff
@@ -836,11 +836,69 @@ write_files:
         sudo touch $cronTaskPath
         sudo chmod 777 $cronTaskPath
 
-        echo "0 0 * * * /home/wowza/renew-cert.sh
-        " > $cronTaskPath
+        echo "0 0 * * * /home/wowza/renew-cert.sh" >> $cronTaskPath
+        
+        if [[ $HOSTNAME == *"prod"* ]] || [[ $HOSTNAME == *"stg"* ]]; then
+           echo "10 0 * * * /home/wowza/check-cert.sh" >> $cronTaskPath
+        fi
 
         sudo -u wowza bash -c "crontab $cronTaskPath"
+  - owner: wowza:wowza
+    permissions: 0775
+    path: /home/wowza/check-cert.sh
+    content: |
+        #!/bin/bash
+
+        # Project
+        project="${project}" 
+
+        # Set Dynatrace Details.
+        dynatrace_token="${dynatrace_token}"
+        dynatrace_tenant="${dynatrace_tenant}"
+
+        # Java Key Store Details.
+        jksPath="/usr/local/WowzaStreamingEngine/conf/ssl.wowza.jks"
+        wowzaServerPath="/usr/local/WowzaStreamingEngine/conf/Server.xml"
+        jksPass=$(cat $wowzaServerPath | grep -i KeyStorePassword | sed 's/<KeyStorePassword>//' | sed 's/<\/KeyStorePassword>//')
```

Comment:
This can get passed in like the other vars `${certPassword}`

Reply:
I think this is a better way, then we never need to pass in the JKS password and will never need to update in our code, it is always being retrieved from the Server file which should always have the correct password.

Reply:
its already always passed in here >
https://github.com/hmcts/cvp-audio-ingress/blob/master/modules/wowza/main.tf#L265

and  used:
https://github.com/hmcts/cvp-audio-ingress/blob/master/cloudconfig.tpl#L27
https://github.com/hmcts/cvp-audio-ingress/blob/master/cloudconfig.tpl#L159
https://github.com/hmcts/cvp-audio-ingress/blob/master/cloudconfig.tpl#L859